timereport/app.py

```diff
@@ -125,6 +124,36 @@ def index():
 
     if action == "list":
         get_by_user = get_user_by_id(f'{backend_url}/user', user_id)
+        if 'total' in params:
+            try:
+                validate_year(params[1])
```

Comment:
Remove functions and just try to create a date object
`date = datetime.strptime(params[1:3], '%Y-%m')`